components/external-secrets-operator/base/subscription.yaml

```
@@ -0,0 +1,31 @@
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: external-secrets-operator
+  annotations:
+    argocd.argoproj.io/sync-wave: "-3"
+spec:
+  targetNamespaces:
+  - external-secrets-operator
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: external-secrets-operator
+  annotations:
+    argocd.argoproj.io/sync-wave: "-2"
+spec:
+  channel: stable
+  installPlanApproval: Manual
```

I know you copied this but we wanted to change this in infra-deployment and we never did, let's make this one automatic: 
```suggestion
  installPlanApproval: Automatic
```